ros/src/computing/perception/detection/packages/cv_tracker/nodes/range_fusion/range_fusion.cpp

```diff
@@ -88,10 +88,32 @@ int main(int argc, char **argv)
 	ros::init(argc, argv, "range_fusion");
 
 	ros::NodeHandle n;
-
-	ros::Subscriber image_obj_sub = n.subscribe("image_obj", 1, DetectedObjectsCallback);
+	ros::NodeHandle private_nh("~");
+
+	std::string image_topic;
+	std::string points_topic;
+	if (private_nh.getParam("image_node", image_topic))
+    	{
+        	ROS_INFO("Setting image node to %s", image_topic.c_str());
+    	}
+	else
+	{
+		ROS_INFO("No image node received, defaulting to image_obj, you can use _image_node:=YOUR_TOPIC");
+		image_topic = "/image_obj";
```

Comment:
Original code uses relative topic path `image_obj` however you use absolute topic path `/image_obj`.
Is this correct ?